python/kfserving/kfserving/storage.py

```diff
@@ -1,25 +1,28 @@
 import logging
 import tempfile
+import os
 
 _GCS_PREFIX = "gs://"
 _S3_PREFIX = "s3://"
-_LOCAL_PREFIX = "/"
+_LOCAL_PREFIX = "file://"
 
 
 class Storage(object):
     @staticmethod
     def download(uri: str) -> str:
         logging.info("Copying contents of %s to local" % uri)
         if uri.startswith(_LOCAL_PREFIX):
```

Comment:
Looks like we're still missing the os.path.exists case

Reply:
Done, Thanks.